manifest.json

```diff
@@ -1,8 +1,8 @@
 {
     "spark_version": "2.2.1",
     "default_spark_dist": {
-        "hadoop_version": "2.6",
-        "uri": "https://downloads.mesosphere.com/spark/assets/spark-2.2.1-2-bin-2.6.tgz"
+        "hadoop_version": "2.7",
+        "uri": "svt-dev.s3.amazonaws.com/spark/spark-2.2.1-bin-2.7.3-dcos-38138.tgz"
```

Comment:
Once PR is in good shape and approved, I'll update the `downloads.mesosphere` uri to include the updated tgz